### DISCUSSION TITLE
[Run issue]:how to get a pdd curve

### DISCUSSION THREAD

**Discussion by bradyhu78** [OpenTOPAS/OpenTOPAS#29] [Run-Problem] [answered]
### What was the issue?

how to get a pdd curve when run mvlinac example? i just get a straight line rather than a  pdd curve

### TOPAS category

Scoring

### Version

3.7

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by icmeyer** [comment]
Hi,
Could you give more information on your error? Which example parameter file are you running? Can you attached your resulting scoring file? 
Thanks!

**Reply by bradyhu78** [reply]
Hello, 
1.First I changed some parameters in WaterPhantom.txt in order to get pdd data ,see the the first attached picture
![WaterPhantom](https://github.com/OpenTOPAS/OpenTOPAS/assets/34639804/3764d7e8-35c3-4f54-856b-3bd56d3cc09b)

2.Then I run MainTxHead.txt with uncomment includeFile = Field.txt. which use the PhaseSPace File 
3.Finanlly I got the Dose file and display the data with Excel Chart, it looks like a straight line, see the the second attached picture 
![Dose Display](https://github.com/OpenTOPAS/OpenTOPAS/assets/34639804/3ecbfcce-02b6-4027-a309-49dfe14d928a)

why it is not a pdd curve? if i want to get a pdd curve,  What should be changed in the txt file?

**Reply by bradyhu78** [reply]
![WaterPhantom](https://github.com/OpenTOPAS/OpenTOPAS/assets/34639804/0e87ed39-9af3-42af-b97d-51908dc1ba14)
sorry, the first picture is this one

**Reply by JoseRamosMendez** [reply, marked as answer]
Hi there, 
The straight line is the binning number. You should plot column 4 vs column  3, instead of both simultaneously; or just column 4 for having a quick display.
Best.

**Reply by bradyhu78** [reply]
oh ,I  got it , Thank you very much!!!
